en/components/buttonGroup.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Button Group Component
+_description: Provides button group functionality to developers and also allows horizontal/vertical alignment, single/multiple selection with toggling.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Button Group components, Angular Button Group controls, Angular Buttons components, Angular Buttons controls
+---
+
+## Button Group
+
+The **igx-ButtonGroup** component provides a button group functionality to developers and also allows horizontal/vertical alignment, single/multiple selection along with toggling. The igx-ButtounGroup component makes use of the [igxButton directive.](button.md)
```

Comment:
I still don't think that you can link to markdown files on production.  Markdown is compiled into HTML, right?

Reply:
@ig-ealbert Yes, .md is compiled into .html, the compiler will create an <a> tag out of this - "[igxButton directive.](button.md)". This is the correct way of linking between .md files.